pwiz_tools/Skyline/Model/Serialization/DocumentWriter.cs

```diff
@@ -878,6 +878,7 @@ private void WriteTransitionChromInfo(XmlWriter writer, TransitionChromInfo chro
                 writer.WriteAttribute(ATTR.retention_time, chromInfo.RetentionTime);
                 writer.WriteAttribute(ATTR.start_time, chromInfo.StartRetentionTime);
                 writer.WriteAttribute(ATTR.end_time, chromInfo.EndRetentionTime);
+                writer.WriteAttributeNullable(ATTR.ccs, chromInfo.IonMobility.CollisionalCrossSectionSqA);

```

Comment:
You also need to add code to read this in in "DocumentReader.TransitionInfo.ReadTransitionPeak".

Also, you need to add a field to the "TransitionPeak" protocol buffer in "pwiz_tools\Skyline\ProtocolBuffers\SkylineDocument.proto".
And then the code to read and write this will go in "TransitionDocNode.GetTransitionPeakProtos" and "TransitionChromInfo.FromProtoTransitionResults".

(Once I finish PR#2405, tests will fail if things like this are missing)

Reply:
I'll do the protocol buffer thing, thanks.  

But I don't understand your comment about ReadTransitionPeak - isn't that happening at line 492?

Reply:
Oops. Yes. You're right. I didn't see that line.

Reply:
@nickshulman ok, protobuf stuff done. Wasn't sure if order mattered in adding that new field (that is, add to the end, or to the end of the numbered fields area?)